src/library/base/man/NA.Rd

```diff
@@ -122,8 +123,11 @@ is.na(x) <- value
   on how methods can be tuned to deal with missing values.
 }
 \examples{
-is.na(c(1, NA))        #> FALSE  TRUE
-is.na(paste(c(1, NA))) #> FALSE FALSE
+is.na(c(1, NA, NA_real__, NaN))        #> FALSE  TRUE  TRUE  TRUE
+is.na(paste(c(1, NA, NA_real__, NaN))) #> FALSE FALSE FALSE FALSE
```

Comment:
?? I think `NA_real_` should only have one underscore at the end? (otherwise "object 'NA_real__' not found") ?

Reply:
typo, fixed